packages/@okta/vuepress-site/docs/guides/set-up-event-hook/implement-your-service/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Implement Your Service
+---
+
+You need to implement a web service with an Internet-accessible endpoint to receive event hook calls from Okta. It's your responsibility to develop the code and to arrange its hosting on a system external to Okta. Okta defines the REST API contract for the REST requests it sends to your service.
```

Comment:
This first sentence is a bit jarring right now. Maybe add a few more words such as "Before you start..." or "This guide assumes that you are interested in..." or something along those lines. 

Also "Event Hook" should be capitalized.

Reply:
Changed to "Before you start....". Corrected capitalization of Event Hook.